data/strings/strings.txt

```diff
@@ -13293,7 +13295,7 @@
   [osm_presentation]
     comment = Text in About and OSM Login screens. %@ is replaced by a local, human readable date.
     tags = android
-    en = Cartographic data from OpenStreetMap as of %@. OSM is like Wikipedia for maps, where you can add or edit places for all users around the world.
+    en = Cartographic data from OpenStreetMap as of %@. OSM is like Wikipedia for maps, where you can add & edit places for users around the world.
```

Comment:
Do not use & in normal text.  Use "and" instead.  "&" is only appropriate in a few contexts.

Reply:
Is there a preference for using & in place of and in straightforward text in en_GB?   To my en_US eyes it comes across as informality (to the point of being sloppy) creeping into text that should be Standard Written English.  However, <rant>I'm fairly old school and growing up I read style guides form the 1950s, so I'm on the prescriptive grammer side somewhat, thinking it is best to do what the kernel of the educated elders thinks is correct, rather than going along with the modern state of television and social media.</rant>

Reply:
Honestly, not sure. i'll change it to 'and'. To my knowledge it's fairly normal, but that could just be because my generation grew up with the internet 😅